www/css/style.css

```diff
@@ -210,9 +223,11 @@ button.button.ng-binding i.icon.ion-edit {
   font-size: 12px;
 }
 button.button.back-button.buttons.button-clear.header-item {
-  color: #fff;
+  color: #303030; opacity: 0.7;
```

Comment:
next week: move colors into a separate css file with names? That way, it will be easier to reuse colors and have a consistent theme